setup.py

```diff
@@ -1,3 +1,7 @@
+# -*- coding: utf-8 -*-
+
+__version__ = '0.4.dev2'
```

Comment:
Please do not move the version information into `__version__`, this is unnecessary and not a convention we use for Pylons Projects.